.github/workflows/ci.yml

```diff
@@ -359,6 +359,7 @@ jobs:
           nginx-crt: ${{ contains(matrix.images.image, 'nap') && secrets.NGINX_AP_CRT || secrets.NGINX_CRT }}
           nginx-key: ${{ contains(matrix.images.image, 'nap') && secrets.NGINX_AP_KEY || secrets.NGINX_KEY }}
           azure-ad-secret: ${{ secrets.AZURE_AD_AUTOMATION }}
+          go_md5: ${{ needs.checks.outputs.go_code_md5 }}
```

Comment:
lint issue?

Reply:
Thanks @vepatel addressed in latest commit